### PR TITLE
[105] Add table titles

### PR DESCRIPTION
**Pull request by przemyslawhaptas**

### CODE REVIEW

**Comment by przemyslawhaptas** [comment]
[[105] nie da się dodac w tabeli w results headerow waitlist, accepted, rejected; bo sa generowane z tym samym elementem thead](https://trello.com/c/KKUrevoM/59-105-nie-da-sie-dodac-w-tabeli-w-results-headerow-waitlist-accepted-rejected-bo-sa-generowane-z-tym-samym-elementem-thead)

**Comment by przemyslawhaptas** [comment]
@szynszyliszys Is it enough to style it?

File: app/views/submissions/_result_table.html.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<div><%= table_title %></div>
```
Comment (lesniakania) [inline app/views/submissions/_result_table.html.erb]: probably it should be `h`?

Comment (przemyslawhaptas) [inline app/views/submissions/_result_table.html.erb]: Probably, that's just a placeholder for now. :D

**Pull request merged**